modules/auxiliary/gather/windows_secrets_dump.rb

```diff
@@ -457,13 +478,13 @@ def print_secret(name, secret_item)
       secret = "dpapi_machinekey: 0x#{machine_key.unpack('H*')[0]}\ndpapi_userkey: 0x#{user_key.unpack('H*')[0]}"
     elsif upper_name.start_with?('$MACHINE.ACC')
       md4 = OpenSSL::Digest::MD4.digest(secret_item)
-      machine, domain = get_machine_name_and_domain
-      print_name = "#{domain}\\#{machine}$"
+      machine, _domain, dns_domain_name = get_machine_name_and_domain_info
+      print_name = "#{dns_domain_name}\\#{machine}$"
       ntlm_hash = "#{Net::NTLM.lm_hash('').unpack('H*')[0]}:#{md4.unpack('H*')[0]}"
       secret_ary = ["#{print_name}:#{ntlm_hash}:::"]
       credential_opts = {
         realm_key: Metasploit::Model::Realm::Key::ACTIVE_DIRECTORY_DOMAIN,
-        realm_value: domain
+        realm_value: dns_domain_name
```

Comment:
I've updated this `dns_domain_name` here, because that's the convention that kerberos for windows requires when authenticating

```
[1] pry(#<Msf::Modules::Auxiliary__Gather__Windows_secrets_dump::MetasploitModule>)> _domain
=> "ADF3"
[2] pry(#<Msf::Modules::Auxiliary__Gather__Windows_secrets_dump::MetasploitModule>)> dns_domain_name
=> "adf3.local"
```

This change also impacts the `creds` view:

```
msf6 auxiliary(gather/windows_secrets_dump) > creds
Credentials
===========

host             origin           service            public                                    private                                                                                              realm       private_type  JtR Format
----             ------           -------            ------                                    -------                                                                                              -----       ------------  ----------
192.168.123.13   192.168.123.13   445/tcp (smb)      fake_computer                             des-cbc-crc:a84af2ec4ff45115                                                                         adf3.local  Krb enc key   
192.168.123.13   192.168.123.13   445/tcp (smb)      Administrator_history10                   aad3b435b51404eeaad3b435b51404ee:32ede47af254546a82b1743953cc4950                                    adf3.local  NTLM hash     nt,lm
1
```